eng/pipelines/templates/dailytests.yml

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Do we need this extra template? Should it go into the apex tests on windows daily file.

@NuGet/nuget-client can someone with more CI authoring experience provide suggestions here?

Reply:
FWIW, I agree. Our primary pipeline was split into many files because there are many (6 or 7?) stages. It used to be one long file, and IMO it was very difficult to manage, because it was so long. But this daily tests pipeline is a single stage, so I don't see the benefit in splitting it up.

Reply:
Removed the extra template _eng/pipelines/templates/dailytests.yml_ yml file, thanks.